rest_framework-stubs/serializers.pyi

```diff
@@ -179,6 +179,7 @@ class ListSerializer(
         min_length: int | None = ...,
         max_length: int | None = ...,
     ) -> None: ...
+    def run_child_validation(self, data: Any) -> Any: ...
```

Comment:
It has the same signature as https://github.com/typeddjango/djangorestframework-stubs/blob/d9d6fd893a654852af5292f4c2de859064777c5e/rest_framework-stubs/fields.pyi#L120

Code: https://github.com/encode/django-rest-framework/blob/9864c47018238dd9f8477df0e4ab2ef62a24fd2b/rest_framework/serializers.py#L653